README.md

```diff
@@ -6,19 +6,49 @@
     ├── <project>                       # Example project name e.g. mobile app
     │   ├── jobs                        # Job DSL code
     │   ├── resources                   # Resources specific to this project
-    ├── <other_project>                 # Another project name e.g. analytics
+    ├── <other_project>                 # Another project name e.g. platform
     │   ├── jobs                        # Job DSL code
     │   ├── resources                   # Resources specific to this project
     ├── <...>                           # A folder for every project!
-    ├── shared                          # Code shared between each project
+    ├── src                             # Code shared between each project
     │   ├── main
     │   │   ├── groovy                  # support classes
     │   │   └── resources
     │   │       └── idea.gdsl           # IDE support for IDEA
     │   └── test
-    │       └── groovy                  # specs
+    │       └── groovy                  # specs (organized by project)
+    │           └──<project>
+    │           └──<other_project>
     └── build.gradle                    # build file
     └── seeds 
         └── <example>.groovy            # Groovy script to create jobs for an individual machine
         └── <other>.groovy              # Groovy script to create jobs for an individual machine
 
+
+## Testing
+
+### Spec Testing
+DSL jobs can be tested using the [Spock test framework](http://spockframework.github.io/spock/docs/1.0/index.html). 
+
+To run all of the test specs:
+`./gradlew test`
+To run all of the test specs within a given project:
+`./gradlew test --tests sample.*`
+To run all a single test spec:
+`./gradlew test --tests sample.SampleJobSpec`
+
+### Linting
```

Comment:
nit: how about we call it static code analysis. that's more like what codenarc does, but without the baggage of sounding like a big nitter.